### DISCUSSION TITLE
Using label to trigger calendar and how to show START/END time at the same time

### DISCUSSION THREAD

**Discussion by JobberRT** [fomantic/Fomantic-UI#2335] [Elements (button, image, label, loader, ...)] [answered]
Hi, I got two question: 
1. How to use a ```Label``` to trigger a ```calendar```?
2. How to set a ```calendar``` to show both START and END selection(like the picture below)? Is this feature haven't been implemented yet?
<img width="602" alt="image" src="https://user-images.githubusercontent.com/17498767/164959139-e29596a0-ba74-42e8-860c-e6df95bbe304.png">

For the first question, I got a wrong example in [JSFiddler](https://jsfiddle.net/9k0x6yve/1/), which I referenced the [Button trigger](https://fomantic-ui.com/modules/calendar.html#button). As you can see, this example will only OPEN the calendar, and will NOT CLOSE the calendar when finished selecting or click other position on the page.

I would like the achieve something like the [Button trigger](https://fomantic-ui.com/modules/calendar.html#button): No ```input``` is needed or the ```input``` could be ```hidden```. In my case, this ```calendar``` is located in a ```divider```, and I'll use ```onChange``` to call AJAX and update the data.

**Comment by lubber-de** [comment, marked as answer]
1. You have to wrap the `ui label` inside a `ui calendar`. I adjusted your jsfiddle accordingly here https://jsfiddle.net/lubber/a0pmesfr/9/  
2. This is currently not implemented, you have to use [two range calendars](https://fomantic-ui.com/modules/calendar.html#range) for now to somehow achieve this. This already got a feature request at #2223
